src/coprocessor/endpoint.rs

```diff
@@ -64,6 +64,8 @@ pub struct Endpoint<E: Engine> {
     /// The soft time limit of handling Coprocessor requests.
     max_handle_duration: Duration,
 
+    slow_log_threshold: f64,
```

Comment:
Using `Duration` type, similar to `max_handle_duration` is better

Reply:
ok